keras_cv/datasets/pascal_voc/segmentation_test.py

```diff
@@ -0,0 +1,285 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import pathlib
+import sys
+
+import tensorflow as tf
+from absl import flags
+
+from keras_cv.datasets.pascal_voc import segmentation
+
+
+class PascalVocSegmentationDataTest(tf.test.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.tempdir = self.get_tempdir()
+        # Note that this will not work with bazel, can need to be rewrite into rely on
```

Comment:
maybe reformat this sentence a little bit

Reply:
Done.